consensus/polybft/README.md

```diff
@@ -104,36 +104,54 @@ It has a native support for running bridge, which enables running cross-chain tr
     --stake-manager <address_of_StakeManager_contract> --stake-token <address_of_erc20_token_used_for_staking>
     ```
 
-11. Finalize genesis validator set on rootchain (SupernetManager) contract. This is done after all validators from genesis do initial staking on rootchain, and it's a final step that is required before starting the child chain. This needs to be done by the deployer of SupernetManager contract (the user that run the deploy command). He can use either its hex encoded private key, or data-dir flag if he has secerets initialized. If enable-staking flag is provided, validators will be able to continue staking on rootchain. If not, genesis validators will not be able update its stake or unstake, nor will newly registered validators after genesis will be able to stake tokens on the rootchain. Enabling of staking can be done through this command, or later after the child chain starts.
+11. Do mint and premine for relayer node. **These commands should only be executed if non-mintable erc20 token is used**
+
+    ```bash
+    $ polygon-edge bridge mint-erc20 \
```

Comment:
The same here. Shouldn't we use the `hydra` executable?

Reply:
These are docs inherited by the upstream and we will address them on a later stage.